pennylane/ops/op_math/sprod.py

```diff
@@ -193,3 +197,26 @@ def _queue_category(self):  # don't queue scalar prods as they might not be Unit
         Returns: None
         """
         return None
+
+    @property
+    def arithmetic_depth(self) -> int:
+        return 1 + self.base.arithmetic_depth
+
+    def simplify(self, depth=-1) -> Operator:
+        if depth == 0:
+            return self
+        if isinstance(self.base, SProd):
+            scalar = self.scalar * self.base.scalar
+            if scalar == 1 and (depth > 1 or depth == -1):
+                return self.base.base.simplify(depth=depth - 2)
+            return SProd(scalar=scalar, base=self.base.base.simplify(depth=depth - 1), id=self.id)
+        if isinstance(self.base, Sum):
+            simplified_sum = self.base.simplify(depth=depth)
+            return Sum(
+                *(  # TODO: Should we assign the same ID to all SProd classes?
```

Comment:
I think we can ignore `id` for the simplify process.  I'm not actually quite sure what the use cases for `id` are. But currently in PennyLane, expansion and transformation processes tend to ignore it.  I think simplify should have that same behavior.

Reply:
The `id` is used by developers/users who want to name a specific operation, and track it later in the process. It is used quite a bit by the Fourier module.

However, we never really worked out what makes the most sense if an operation with id is decomposed

Reply:
`id` ignored! 👍